ui.apps/src/main/content/jcr_root/apps/aemdemo/components/like/clientlib-like/css/like.css

```diff
@@ -0,0 +1,35 @@
+.like_div{
+    font-size: 50px;
+    color: grey;
+    display: inline-block;
+}
+
+.like_button, .dislike_button{
+    width: 50px;
+    height: 50px;
+    background-image: url(/content/dam/aemdemo/like.jpg);
+    background-size: cover;
+    border: none;
+    border-radius: 60%;
+    display: inline;
+}
+
+.dislike_button{
+    background-image: url(/content/dam/aemdemo/dislike.jpg);
```

Comment:
That's OK!
Just be informed, that somewhat of a current trend, is to use SVG instead of raster images, wherever possible. 
SVG's are lightweight, suffer no quality degradation, can be placed inline in HTML, can be styled, etc. Of course, the exclusion is photographs. You cannot render a photo in SVG)